app/src/main/java/com/example/saar/EventAdapter.java

```diff
@@ -44,10 +51,24 @@ public TimeLineViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(TimeLineViewHolder holder, int position) {
+    public void onBindViewHolder(final TimeLineViewHolder holder, final int position) {
```

Comment:
why final?

Reply:
If not final, then we cannot use holder and position inside onclick()